src/pages/instances/CreateInstanceForm.tsx

```diff
@@ -334,15 +372,17 @@ const CreateInstanceForm: FC = () => {
                   isSubmitting={formik.isSubmitting}
                   isDisabled={!formik.isValid || !formik.values.image}
                   buttonLabel="Create"
-                  appearance="default"
+                  appearance={isLocalIsoImage ? "positive" : "default"}
                   onClick={() => submit(formik.values, false)}
                 />
-                <SubmitButton
-                  isSubmitting={formik.isSubmitting}
-                  isDisabled={!formik.isValid || !formik.values.image}
-                  buttonLabel="Create and start"
-                  onClick={() => submit(formik.values)}
-                />
+                {!isLocalIsoImage && (
+                  <SubmitButton
+                    isSubmitting={formik.isSubmitting}
+                    isDisabled={!formik.isValid || !formik.values.image}
+                    buttonLabel="Create and start"
+                    onClick={() => submit(formik.values)}
+                  />
+                )}
```

Comment:
I'm not sure I understand why we need to force the users to start the instance manually in this case. They would still need to start it in order to do... anything. So why removing the possibility to "create and start"?

Reply:
The user needs to be on the graphical console and finish the installation for the VM to work. Starting the VM here would imply the instance is usable after, which it is not. This is especially critical for windows, where you need to "press any key" to boot from the mounted ISO within a 5 second interval, and not from the empty disk. Starting the VM will bring the VM into a failed boot state.

So we decided to disable the "Create and start" option and have the changed notification that links to the console. So the users will likely go to the console, start it there and naturally understand to follow the installation process via the GUI.

Reply:
We considered changing the redirect target to the console, but went against it, because it might be annoying to have different redirect targets for experienced users, that have an expectation of where they land on launch.